src/docs/background/23-how-capabilities-work/index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: How Capabilities work?
+slug: how-capabilities-work
+updated: 2019-12-19
+category: background
+ingress:
+  Capabilities framework offers tools to manage regulatory aspects of
+  provider onboarding. This article outlines mandatory changes to the
+  framework and instructions regarding your actions on reacting to those
+  changes.
+published: true
+---
+
+## Introduction
+
+Coping with regulatory environment regarding provider onboarding can
+be a complex and daunting endeavour. Fulfilling these requirements
+means that you need to be collecting a variety of information from
+your providers in order to be able to process payments. Stripe has a
+mechanism for controlling the amount of information collected from providers called
+[Capabilities](https://stripe.com/docs/connect/capabilities-overview).
+
+Capabilities are a collection of settings that can be requested for
+each provider. What Capabilities are required determines what
+information Stripe requires to be collected from the providers.
+
+## Changes to Capabilities framework
+
+Stripe is introducing mandatory changes to the Capabilities framework.
+In the beginning of 2020, Stripe will enforce that Capabilities are
+required for all accounts, existing and new. Stripe’s deadlines for
+reacting to these changes are:
+- 20th of February, 2020 for customers in the US.
+- 31st of March, 2020 for the rest of the world.
+- In practice, there might be small variations for these depending on the
+ country and activity of the marketplace.
+
+After requesting new Capabilities, your providers need to most likely
+provide additional verification information to Stripe and you need to
+update your provider onboarding and payment information updating
+processes in your marketplace application.
+
+## How should you handle the changes to Capabilities framework?
+
+For Flex payments to operate smoothly, your accounts need to have the
+`card_payments` and `transfers` Capabilities required and activated.
+The `card_payments` will enable the providers to accept payments by
+card and the `transfers` will allow the platform account to move money
+to connected accounts. Requesting `card_payments` will enforce also
+requesting the `transfers` capability.
+
+Requesting Capabilities can be done during [Stripe
+Account](https://www.sharetribe.com/api-reference/marketplace.html#stripe-account)
+[create](https://www.sharetribe.com/api-reference/marketplace.html#create-stripe-account)
+and
```

Comment:
It is really hard to visually separate the two links next to each other, party because our hover effect is really subtle. Maybe the "Stripe account" link can be removed or just combined to the following "create" link?